JsonFormReadme.md

```diff
@@ -0,0 +1,61 @@
+
+-----------------------------------------------------------------------------------------------
+# documentation   
+
+## required attributes
+
+1) key - a key is always string and enclosed in quotation mark
+
+2) name - specify the name of a feild
+
+3) label - used to create caption for a form control
+
+4) id - specify the unique id for an element
+
+5) type - defines the feild
+
+6) placeholder - shows information belongs in a feild
+
+7) validation - this value is a function that return a string errort or null if the feild is valid
+
+8) validation_server - it needs to validate on the server in addtion to on the client
```

Comment:
@sajana1829 If you understand the code then you will see that `validation_server` is only working on the server. For example, `exists` validation will only work on server because it checks if the value exists in the database. Since only server has direct access to database, that validation will only work on the server.